const errors = require('./src11/errors');
const handle = require('./src11/handling');

//errors.standardError.range();

/*try {
    errors.standardError.range();
} catch(err) {
    console.log('Ha ocurrido un error en la creaci√≥n del array', err);
}*/

//handle.errorFirstCallbackWrong();